### PR TITLE
Add tests for pkg/network package

### PR DESCRIPTION
**Pull request by cclerget**
**Description of the Pull Request (PR):**

- #2559

**Before submitting a PR, make sure you have done the following:**

- Read the [Guidelines for Contributing](../CONTRIBUTING.md), and this PR conforms to the stated requirements.
- Added changes to the [CHANGELOG](../CHANGELOG.md) if necessary according to the [Contribution Guidelines](../CONTRIBUTING.md)
- Added tests to validate this PR and tested this PR locally with a `make testall`
- Based this PR against the appropriate branch according to the [Contribution Guidelines](../CONTRIBUTING.md)
- Added myself as a contributor to the [Contributors File](../CONTRIBUTORS.md)

Attn: @singularity-maintainers

### CODE REVIEW

File: pkg/network/network.go [hunk 1]
```diff
@@ -294,7 +269,7 @@ func (m *Setup) SetArgs(args []string) error {
 
 				splittedPort := strings.SplitN(value, "/", 2)
 				if len(splittedPort) != 2 {
-					return fmt.Errorf("badly formatted portmap argument '%s', must be of form portmap=hostPort:containerPort/protocol", splitted[1])
+					return fmt.Errorf("badly formatted portmap argument '%s', must be of form portmap=hostPort:containerPort/protocol", splitted)
```
Comment (mem) [inline pkg/network/network.go]: Wouldn't it be more clear if you use the original value here (arg)?

**Review by mem — COMMENTED** [review]
Tests look good to me.

File: pkg/network/network.go (hunk 1 above)
Comment (cclerget) [reply pkg/network/network.go]: @mem arg could contain arguments not related to portmap, finally changed for `value` to display the argument value generating the error

Comment (mem) [reply pkg/network/network.go]: Ok

**Review by bauerm97 — APPROVED** [review]

**Pull request merged**